frontend/src/pages/RunList.tsx

```diff
@@ -47,14 +46,15 @@ interface RecurringRunInfo {
 interface DisplayRun {
   experiment?: ExperimentInfo;
   recurringRun?: RecurringRunInfo;
-  run: ApiRun;
+  run: V2beta1Run;
   pipelineVersion?: PipelineVersionInfo;
   error?: string;
 }
 
 interface DisplayMetric {
   metadata?: MetricMetadata;
-  metric?: ApiRunMetric;
+  // run metric field is currently not supported in v2 API
```

Comment:
I think we need to hide it from official documentation as well. The change here is going to make metrics disappear on both V1 and V2. If we decide to hide metrics for both V1 and V2, can you go update https://www.kubeflow.org/docs/components/pipelines/v1/sdk/pipelines-metrics/#view-the-metrics to hide this feature as well?